lib/config.ts

```diff
@@ -715,6 +715,10 @@ export interface Config {
   nodeDebug?: boolean;
   debuggerServerPort?: number;
   frameworkPath?: string;
+
+  /**
+   * Deprecated: Element explorer is no longer supported.
```

Comment:
Maybe expand this: 'Element explorer depends on the WebDriver control flow, and thus is no longer supported'

Reply:
Done.